README.md

```diff
@@ -7,6 +7,9 @@
   </a>
   <a href="https://github.com/iv-org/invidious/actions">
     <img alt="Build Status" src="https://github.com/iv-org/invidious/workflows/Invidious%20CI/badge.svg">
+  </a>
+    <a href="https://github.com/iv-org/invidious/commits/master">
```

Comment:
```suggestion
  <a href="https://github.com/iv-org/invidious/commits/master">
```

Reply:
The editor messed up the indentation oops.